src/views/Login.vue

```diff
@@ -54,6 +54,7 @@ export default {
   },
   methods: {
     onSubmit (email, password) {
+      console.log('submit form')
```

Comment:
I think we should delete the logging statements here.

Reply:
Yes certainly!